emranlib

```diff
@@ -11,6 +11,10 @@
 # To make modifications to this file, edit `tools/run_python.sh` and then run
 # `tools/create_entry_points.py`
 
+unset _PYTHON_SYSCONFIGDATA_NAME
+unset PYTHONHOME
+unset PYTHONPATH
```

Comment:
What about the windows .bat file launchers.. shouldn't we do that same there?

(I'm a little worried about windows since I'm not sure how scoping works in .bat files.. if you unset a variables does that unset it in the calling process too?)

Reply:
tbh i personnally don't intend to produce sysconfig data for windows->wasm cross compilation of CPython modules. i'd say just add only -E until somebody has a real problem with python subprocesses.

python-wasm-sdk primary target is ubuntu github CI, someone else would have to test that on windows if they really want to.